monkey/monkey_island/cc/next_ui/src/_components/app-bar/AppBar.tsx

```diff
@@ -0,0 +1,117 @@
+import { useRouter } from 'next/navigation';
+import { ALL_LINKS, AUTHENTICATED_PATHS } from '@/constants/paths.constants';
+import Tooltip from '@mui/material/Tooltip';
+import IconButton from '@mui/material/IconButton';
+import Avatar from '@mui/material/Avatar';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import LogoutButton from '@/_components/logout-button/LogoutButton';
+import React, { useState } from 'react';
+
+const MonkeyAppBar = () => {
+    const router = useRouter();
+
+    // const [anchorElNav, setAnchorElNav] = React.useState<null | HTMLElement>(
+    //     null
+    // );
+
+    const [anchorElUser, setAnchorElUser] = useState<null | HTMLElement>(null);
+
+    // const handleOpenNavMenu = (event: React.MouseEvent<HTMLElement>) => {
+    //     setAnchorElNav(event.currentTarget);
+    // };
+
+    const handleOpenUserMenu = (event: React.MouseEvent<HTMLElement>) => {
+        setAnchorElUser(event.currentTarget);
+    };
+
+    // const handleCloseNavMenu = () => {
+    //     setAnchorElNav(null);
+    // };
+
+    const handleCloseUserMenu = () => {
+        setAnchorElUser(null);
+    };
+
+    const handleRouteClick = (path: string) => {
+        router.push(path);
+    };
+
+    return (
+        <AppBar
+            id={classes['app-bar']}
+            position="static"
+            color="primary"
+            enableColorOnDark>
+            <Container maxWidth="xl">
+                <Toolbar disableGutters>
+                    <Box
+                        className={'app-logo-container'}
+                        onClick={() =>
+                            handleRouteClick(AUTHENTICATED_PATHS.ROOT)
+                        }>
+                        <SvgIcon inheritViewBox={true}>
+                            <MonkeyHeadSvg />
+                        </SvgIcon>
+                        <SvgIcon inheritViewBox={true}>
+                            <MonkeyNameSvg />
+                        </SvgIcon>
+                    </Box>
+                    <Box className={'app-links'}>
+                        {ALL_LINKS.map((currPath) => (
+                            <div
+                                key={currPath.label}
+                                className={'app-route-link'}
+                                onClick={() => handleRouteClick(currPath.path)}>
+                                <Typography>{currPath.label}</Typography>
+                            </div>
+                        ))}
+                    </Box>
+                    <Box className={'profile-wrapper'}>
+                        <Tooltip title="Open settings">
+                            <IconButton
+                                onClick={handleOpenUserMenu}
+                                sx={{ p: 0 }}>
+                                <Avatar
+                                    alt="Remy Sharp"
+                                    src="/static/images/avatar/2.jpg"
```

Comment:
Should we do `import AvatarImage from "/static/images/avatar/2.jpg"`? The problem with hard-coded paths is that they are all over the place.